CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## FEATURES
+
+- Add gaiad snapshots command set ([\#2974](https://github.com/cosmos/gaia/pull/2974))
```

Comment:
Please do not update CHANGELOG.md directly. Just add an the above text to a file in `.changelog/unreleased/features`. See `.changelog/` for examples.

Reply:
Thank you.  Read the instructions incorrectly.  Will fix.